app/views/board.scala.html

```diff
@@ -0,0 +1,44 @@
+@import ttt.game.Marks
+@import ttt.game.Board
+@import ttt.game.GameEngine
+
+@(board: Board, game: GameEngine)
+
+@main("Tic Tac Toe") {
+    <body>
+        <h1>Tic Tac Toe</h1>
+
+        <table id="board">
+        @for((row, index) <- board.getRows.zipWithIndex) {
+            <tr>
+            @for((cell, position) <- row.zipWithIndex) {
+                <th class="cell">
+                @if(cell == Marks.NULL) {
+                    @if(!game.isOver) {
```

Comment:
Can you think of a way to minimising double `if` conditions and more specifically, the check against `Marks.NULL`?